ScipDotnet/ScipIndexer.cs

```diff
@@ -114,9 +117,18 @@ await host.Services.GetRequiredService<MSBuildWorkspace>()
         }
         else
         {
-            var walker = new ScipCSharpSyntaxWalker(doc, semanticModel, options, globals);
+            var symbolFormatter = new ScipSymbolFormatter(doc, options, globals);
             var root = await document.GetSyntaxRootAsync();
-            walker.Visit(root);
+            if (language == "C#")
+            {
+                var walker = new ScipCSharpSyntaxWalker(symbolFormatter, semanticModel);
+                walker.Visit(root);
+            }
+            else
```

Comment:
It might make sense to guard against this here
```suggestion
            else if (language == "Visual Basic")
```